contributing-guides/maintainers-guide.md

```diff
@@ -109,3 +115,10 @@ for the behavior expected of tldr-pages maintainers.
  should be documented as an issue with the [archive](https://github.com/tldr-pages/tldr/issues?q=label%3Aarchive) label
   and must be communicated with other maintainers.
 - All repository/organization settings changes must be documented as an issue with the [archive](https://github.com/tldr-pages/tldr/issues?q=label%3Aarchive) label.
+
+## IV. Handling failing actions and CLA checks
+
+- While merging multiple pull requests at the same time there is a change that the deploy step might fail in the GitHub Actions workflow. In such cases, the maintainer should **re-run** the worklflow of the commit which was last merged (to prevent overwriting of assets).
```

Comment:
```suggestion
- While merging multiple pull requests at the same time there is a chance that the deploy step might fail in the GitHub Actions workflow. In such cases, the maintainer should only **re-run** the workflow of the commit which was last merged (to prevent overwriting of assets by previous commits).
```